templates/taxi/car_list.html

```diff
@@ -3,20 +3,21 @@
 {% block content %}
   <h1>
     Car list
-    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
+    <span class="position-relative pl-5">
+      <a href="{% url 'taxi:car-create' %}" class="btn btn-primary position-relative ml-5">Create New car</a>
+    </span>
   </h1>
   {% if car_list %}
-    <ul>
+    <ul class="list-group">
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+        <li class="list-group-item">
+          <a class="btn btn-outline-primary btn-sm px-4 mr-2" href="{% url "taxi:car-detail" pk=car.id %}">id: {{ car.id }}</a>
```

Comment:
The URL tag for 'taxi:car-detail' should use single quotes instead of double quotes to maintain consistency with Django template syntax. It should be `{% url 'taxi:car-detail' pk=car.id %}`.